modules/API/API.cpp

```diff
@@ -634,4 +646,23 @@ void API::ready() {
     }));
 }
 
+void API::wait_evse_manager_ready() {
+    std::unique_lock lock(evse_manager_mux);
+    evse_manager_cv.wait(lock, [this] { return this->evse_manager_ready <= 0; });
+}
+
+void API::notify_evse_manager_ready(bool ready) {
+    if (ready) {
+        bool notify{false};
+        {
+            std::lock_guard lock(evse_manager_mux);
+            evse_manager_ready--;
```

Comment:
These changes rely on the fact that each EvseManager publishes `ready` only once (which is currently given). Instead of using a counter that counts down you could actually check for each EVSE individually (e.g. by using a map).

Reply:
@Pietfried updated the approach so that an EVSE manager can be ready multiple times and not cause an issue.